frontends/api/src/test-utils/factories/user.ts

```diff
@@ -21,4 +21,18 @@ const profile: PartialFactory<Profile> = (overrides = {}): Profile => ({
   ...overrides,
 })
 
-export { profile }
+const user: PartialFactory<User> = (overrides = {}): User => {
```

Comment:
I ended up not using this factory but decided to keep it since we don't have it.